srcpkgs/Chipmunk2D/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'Chipmunk2D'
+pkgname=Chipmunk2D
+version=7.0.2
+revision=1
+wrksrc="Chipmunk2D-Chipmunk-${version}"
+build_style=cmake
+short_desc="Fast and lightweight 2D game physics library"
+maintainer="Ekaterina Vaartis <vaartis@cock.li>"
+license="MIT"
+homepage="http://chipmunk-physics.net/"
+distfiles="https://github.com/slembcke/Chipmunk2D/archive/Chipmunk-${version}.tar.gz"
+checksum="6b6d8d5d910c4442fb9c8c4c46a178126d8c21d075cdb3ce439a7f8d8757b0ca"
+
+configure_args="-DBUILD_DEMOS=NO"
+
+case "$XBPS_TARGET_MACHINE" in
+	*-musl) broken="needs sysctl.h" ;;
```

Comment:
This can be fixed quite easily, see https://github.com/slembcke/Chipmunk2D/pull/171